packages/contracts-bedrock/justfile

```diff
@@ -77,8 +77,23 @@ print-pinned-block-number:
 # TODO(opcm upgrades): unskip the "NMC" tests which fail on the forked upgrade path with "UnexpectedRootClaim" errors.
 prepare-upgrade-env *ARGS : build-go-ffi
   #!/bin/bash
-  echo "Running upgrade tests at block $pinnedBlockNumber"
-  export FORK_BLOCK_NUMBER=$pinnedBlockNumber
+  if [ ! -z "$FORK_BASE_CHAIN" ]; then
+    case "$FORK_BASE_CHAIN" in
+      "mainnet")
+        export FORK_BLOCK_NUMBER=21387830
+        ;;
+      "sepolia")
+        export FORK_BLOCK_NUMBER=7701807
+        ;;
+      *)
+        export FORK_BLOCK_NUMBER=$pinnedBlockNumber
+        ;;
+    esac
+  else
+    export FORK_BLOCK_NUMBER=$pinnedBlockNumber
```

Comment:
I would prefer to see this logic go into `print-pinned-block-number`.

Reply:
The problem here is that I don't have assurances that the FORK_BASE_CHAIN will be set to know which block number to use